pkg/reconciler/cronjobsource/resources/labels.go

```diff
@@ -16,9 +16,16 @@ limitations under the License.
 
 package resources
 
+const (
+	// controllerAgentName is the string used by this controller to identify
+	// itself when creating events.
+	controllerAgentName = "cronjob-source-controller"
+)
+
 // Labels are the labels attached to all resources based on a CronJobSource.
 func Labels(name string) map[string]string {
 	return map[string]string{
 		"sources.eventing.knative.dev/cronJobSource": name,
+		"eventing.knative.dev/source":                controllerAgentName,
```

Comment:
What is the reason this is being added?

Reply:
This done to help prometheus scrape the sources as in [here](https://github.com/knative/eventing/pull/1790/files#diff-6ba5a63d9a5b51bd135af21cb5b37ee0R139). We have a similar label in apiserversource([here](https://github.com/knative/eventing/pull/1790/files#diff-6ba5a63d9a5b51bd135af21cb5b37ee0L122)) hence I added with consistency in mind.